### PR TITLE
Nav updates

### LINKED ISSUE #361: Update Eyebrow Dropdown Specs
Updated dropdown specs need to be added to eyebrow specs. 
17 eyebrow prod nav specs for helixized pilot

### LINKED ISSUE #362: Update Application Nav with new specs
Need to update the redline specs and the image to reflect the following:

Default bg: 			#212121 (Gray 975)
Default text color: 		#d8d8d8 (Gray 400)
Hover bg: 			#424242 (Gray 900)
Hover text color: 		#fafafa (Gray 25)
Selected bg: 			#424242 (Gray 900) - same as hover
Selected text color: 	#fafafa (Gray 25) - same as hover
Disabled hover bg: 	#333333 (Gray 950)
Disabled text color: 	#9e9e9e (Gray 600)

### PR DESCRIPTION
**Pull request by technabors**
Splits the application-navigation.md file into three separate files. Notes below:

- Application-navigation.md renamed to eyebrow.md
- top-nav.md added, content copied from old page
- left-nav.md added, content copied from old page
- New preview images added, adjusted to fit the three page strategy
- New eyebrow content added (WIP)
- Closes #361 
- Closes #362 
- Add empty hidden code pages for top nav and eyebrow.
- Add functional code page for left nav.

Note: None of the content for top-nav or left-nav was adjusted but it appears as 'newly' written code because it was moved from another file. We can probably ignore those files outside of the eye-test in the editorial review process. All content in the eyebrow doc is fresh / newly written so that will need an editorial review.

### CODE REVIEW

File: _component_design/left-nav.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+- This component can extend to fit any navigation information architecture (IA) by scrolling when the list of navigation items exceeds the screen view height. If there are many options to display, the left navigation is scroll-able so that users have access to all items.
+- The preferred implementation includes only primary and secondary navigation links.
+- An alternative, non-preferred implementation includes [tertiary](#tertiary-left-navigation) navigation links.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.cdn_url}}/img/layout/navigation-left.svg)
```
Comment (araiford) [inline _component_design/left-nav.md]: Going forward, remove all the empty space surrounding images in the image file.

File: _component_design/left-nav.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Use the left navigation component for web applications, unless there is a firm implementation constraint that requires top navigation.
+
+- This component can extend to fit any navigation information architecture (IA) by scrolling when the list of navigation items exceeds the screen view height. If there are many options to display, the left navigation is scroll-able so that users have access to all items.
```
Comment (araiford) [inline _component_design/left-nav.md]: Remove this paragraph

File: _component_design/left-nav.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+- The preferred implementation includes only primary and secondary navigation links.
```
Comment (araiford) [inline _component_design/left-nav.md]: add an H3 above these titled "Best practices"

File: _component_design/left-nav.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+## Left navigation
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
```
Comment (araiford) [inline _component_design/left-nav.md]: Add H3 title "When to use"

File: _component_design/left-nav.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+![]({{site.cdn_url}}/img/layout/navigation-left.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Left navigation color and typeface
```
Comment (araiford) [inline _component_design/left-nav.md]: Change to `Specifications`

File: _component_design/left-nav.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
```
Comment (araiford) [inline _component_design/left-nav.md]: add H3 titled `Style`

File: _component_design/left-nav.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+![]({{site.cdn_url}}/img/layout/navigation-left-color-type.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Left navigation spacing
```
Comment (araiford) [inline _component_design/left-nav.md]: remove

File: _component_design/left-nav.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
```
Comment (araiford) [inline _component_design/left-nav.md]: Add H3 titled `Spacing`

File: _component_design/left-nav.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+![]({{site.cdn_url}}/img/layout/navigation-left-spacing-vertical.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+### Tertiary left navigation
```
Comment (araiford) [inline _component_design/left-nav.md]: Change to H2

File: _component_design/left-nav.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
```
Comment (araiford) [inline _component_design/left-nav.md]: Add H3 titled `When to use`

File: _component_design/top-nav.md [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+{% include toc.html %}
+
+
+<section class="static-section" markdown="1">
+
+## Introduction
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
```
Comment (araiford) [inline _component_design/top-nav.md]: Add H3 titled `When to use`

File: _component_design/top-nav.md [hunk 12]
```diff
@@ -0,0 +1,168 @@
...
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Top navigation color and typeface
```
Comment (araiford) [inline _component_design/top-nav.md]: Change to `Style`

File: _component_design/top-nav.md [hunk 13]
```diff
@@ -0,0 +1,168 @@
...
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.cdn_url}}/img/layout/navigation-top.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
```
Comment (araiford) [inline _component_design/top-nav.md]: Add H2 titled `Specifications`

File: _component_design/top-nav.md [hunk 14]
```diff
@@ -0,0 +1,168 @@
...
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Top navigation spacing
```
Comment (araiford) [inline _component_design/top-nav.md]: Change to `Spacing`

File: _component_design/Eyebrow.md [hunk 15]
```diff
@@ -0,0 +1,430 @@
...
+
+### Logo structure
+
+The logo can only be clickable when the application has a default landing page, such as a dashboard or overview.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Logo states when the logo is clickable for navigation"] [class:"image bg-light border"] %}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/Eyebrow.md]: updated.

File: _component_design/left-nav.md [hunk 16]
```diff
@@ -0,0 +1,134 @@
...
+status: stable
+resource: true
+helix-ui-css: true
+helix-ui-javascript: true
+last-modified: 2017-12-11
+---
+
+<section class="static-section" markdown="1">
+
+## Primary navigation
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/left-nav.md]: updated

File: _component_design/left-nav.md [hunk 17]
```diff
@@ -0,0 +1,134 @@
...
+
+Use primary navigation for the main sections of an application, or for drop-down categories that organize sections into higher-level groups.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/left-nav.md]: removed.

File: _component_design/left-nav.md [hunk 18]
```diff
@@ -0,0 +1,134 @@
...
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Left navigation
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/left-nav.md]: removed.

File: _component_design/left-nav.md (hunk 4 above)
Comment (technabors) [reply _component_design/left-nav.md]: updated.

File: _component_design/left-nav.md (hunk 2 above)
Comment (technabors) [reply _component_design/left-nav.md]: removed.

File: _component_design/left-nav.md (hunk 3 above)
Comment (technabors) [reply _component_design/left-nav.md]: added.

File: _component_design/left-nav.md (hunk 5 above)
Comment (technabors) [reply _component_design/left-nav.md]: updated.

File: _component_design/left-nav.md (hunk 6 above)
Comment (technabors) [reply _component_design/left-nav.md]: added.

File: _component_design/left-nav.md (hunk 8 above)
Comment (technabors) [reply _component_design/left-nav.md]: updated.

File: _component_design/left-nav.md (hunk 9 above)
Comment (technabors) [reply _component_design/left-nav.md]: updated.

File: _component_design/top-nav.md (hunk 11 above)
Comment (technabors) [reply _component_design/top-nav.md]: added.

File: _component_design/top-nav.md (hunk 14 above)
Comment (technabors) [reply _component_design/top-nav.md]: added.

File: _component_design/top-nav.md (hunk 12 above)
Comment (technabors) [reply _component_design/top-nav.md]: added.

File: _component_design/top-nav.md [hunk 19]
```diff
@@ -0,0 +1,168 @@
...
+{% figure [caption:"Horizontal padding specs for primary navigation items"] [class:"image bg-light border"] %}
+![]({{site.cdn_url}}/img/layout/navigation-top-spacing-horizontal.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/top-nav.md]: updated.

File: _component_design/top-nav.md [hunk 20]
```diff
@@ -0,0 +1,168 @@
...
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Top navigation drop-down menu color and typeface
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/top-nav.md]: updated.

File: _component_design/top-nav.md [hunk 21]
```diff
@@ -0,0 +1,168 @@
...
+![]({{site.cdn_url}}/img/layout/navigation-dropdown-color-type.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Top navigation drop-down element spacing
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/top-nav.md]: fixed. I used search and replace for the columns and looks like this one didn't fit the parameters.

File: _component_design/left-nav.md (hunk 7 above)
Comment (technabors) [reply _component_design/left-nav.md]: removed.

File: _component_design/left-nav.md (hunk 1 above)
Comment (technabors) [reply _component_design/left-nav.md]: I will remake the images for left-nav in an additional commit today.

File: _component_design/top-nav.md [hunk 22]
```diff
@@ -0,0 +1,171 @@
...
+![]({{site.cdn_url}}/img/layout/navigation-dropdown-color-type.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Spacing specifications
```
Comment (araiford) [inline _component_design/top-nav.md]: BAH!  Sorry about this.  Just noticed this last H2.

Should be an H3 inside the hxCol.

Comment (technabors) [reply _component_design/top-nav.md]: updated

File: _component_design/Eyebrow.md [hunk 23]
```diff
@@ -0,0 +1,430 @@
...
+  The eyebrow is the top menu bar across all breakpoints within the application frame of a Helix interface. The eyebrow provides access to platform-level services and aggregate information across a user's accounts at Rackspace.
+preview-image: preview-images/eyebrow.svg
+status: stable
+resource: true
+helix-ui-css: true
+helix-ui-javascript: true
+last-modified: 2017-08-30
+---
+
+<!-- {% include toc.html %} commented out due to the size of the doc -->
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: So are we starting to skip TOCs?

File: _component_design/Eyebrow.md [hunk 24]
```diff
@@ -0,0 +1,430 @@
...
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Use the eyebrow nav bar to provide access to globally available features, such as account settings, billing, tickets, user management and so on.
+- The eyebrow must not be used for navigating to product features like infrastructure management.
+
+### Best practice
+
+- The <a href="https://one.rackspace.com/display/reach/Pilot+Shared+Navigation">Pilot navigation API</a> is highly recommended for use in UIs which will be presented along side customer facing applications, such as MyRackspace, FAWS, or the Rackspace Cloud Control Panel.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: Change this HTML hyperlink to a Markdown hyperlink.

File: _component_design/Eyebrow.md [hunk 25]
```diff
@@ -0,0 +1,430 @@
...
+
+## Icon nav item
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Structure
+
+- Nav items fill the height of the nav bar, and have 16 pixels of left and right padding.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: s/`16 pixels`/`16px` for consistency across the site

File: _component_design/Eyebrow.md [hunk 26]
```diff
@@ -0,0 +1,430 @@
...
+- The icon is 16 pixels, centered inside an 18 pixel space to facilitate better alignment for icons with different width and height ratios.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: ditto

File: _component_design/Eyebrow.md [hunk 27]
```diff
@@ -0,0 +1,430 @@
...
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### States
+
+#### Hover & open
+- Hover state is also used as the “Open” state for nav items that spawn a dropdown.
+
+#### Disabled items
+
+- Disabled items receive the cursor: not-allowed style, to give an additional indicator that the user cannot access the hovered item.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: s/`the cursor: not-allowed style`/```the `cursor: not-allowed` style```

File: _component_design/Eyebrow.md [hunk 28]
```diff
@@ -0,0 +1,430 @@
...
+
+## Drop-down nav
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Structure
+
+- Nav items fill the height of the nav bar, and have 16 pixels of left and right padding.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: see note re: pixels vs px

File: _component_design/Eyebrow.md [hunk 29]
```diff
@@ -0,0 +1,430 @@
...
+- The icon is 16 pixels, centered inside an 18 pixel space to facilitate better alignment for icons with different width and height ratios.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: ditto

File: _component_design/Eyebrow.md [hunk 30]
```diff
@@ -0,0 +1,430 @@
...
+- Icon nav items can spawn drop-down menus or just navigate the user directly to a new area.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Username construction           "] [class:"image bg-light border"] %}
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: Remove the extra whitespace in the caption.

File: _component_design/Eyebrow.md [hunk 31]
```diff
@@ -0,0 +1,430 @@
...
+
+### States
+
+#### Hover & open
+
+- Hover state is also used as the “Open” state for nav items that spawn a dropdown.
+
+#### Disabled state
+
+- Disabled state receives the cursor: not-allowed style, to give an additional indicator that the user cannot access the hovered item.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: see note above re:code marking

File: _component_design/Eyebrow.md [hunk 32]
```diff
@@ -0,0 +1,430 @@
...
+## Drop-down menu
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Use when the eyebrow item offers navigation to more than one destination.
+- Drop-down menus can be safely used with Icon Nav Items.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: lowercase for "Icon Nav Items" (check elsewhere in this doc, too.)

File: _component_design/Eyebrow.md [hunk 33]
```diff
@@ -0,0 +1,430 @@
...
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+    
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Structure
+
+- The drop-down menu has the same background color as the nav bar that spawned it.  In this case, the background is black: `#000000` (Gray 1000).
+- Menu’s can be divided into sections to enhance readability.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: s/`Menu's`/`Menus`

File: _component_design/Eyebrow.md [hunk 34]
```diff
@@ -0,0 +1,430 @@
...
+- The top section has 20 pixels of space on top, with 16 pixels of space on bottom.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: see note re:pixels vs px

Comment (nimbinatus) [reply _component_design/Eyebrow.md]: for the rest of the page, too

File: _component_design/Eyebrow.md (hunk 25 above)
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: Throughout, spell out `navigation` instead of `nav`.

**Comment by araiford** [comment]
LGTM

File: _component_design/Eyebrow.md [hunk 35]
```diff
@@ -0,0 +1,430 @@
...
+
+<div class="hxRow"  markdown="1">
+    
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Specifications
+
+- Use key-value items when there is a need to present important info at a high level, such as account number in the account dropdown, or timezone in the user dropdown.
+- Key-value items in the menu are not navigable, meaning that they cannot be clicked on to navigate, and they do not feature interactive states.
+- Key value items should be selectable using the cursor, so that the user may copy and paste the info, such as account number.
```
Comment (nimbinatus) [inline _component_design/Eyebrow.md]: s/`key value`/`key-value`

File: _component_design/left-nav.md [hunk 36]
```diff
@@ -0,0 +1,137 @@
...
+<section class="static-section" markdown="1">
+
+## Tertiary left navigation
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+If you have a very large IA, you can use tertiary navigation elements. However, in most cases we recommend that you compress or sort the IA so that only primary and secondary links are needed.
```
Comment (nimbinatus) [inline _component_design/left-nav.md]: Spell out IA

File: _component_design/Eyebrow.md (hunk 23 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: The ToC on this page was really unruly so I omitted it on this doc. This is what prompted the discussion around updating the ToC style / structure in slack last week.

image

File: _component_design/Eyebrow.md (hunk 24 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: ugh, not sure why I did that, some habits are hard to kick. Updated.

File: _component_design/Eyebrow.md (hunk 25 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: updated.

Comment (technabors) [reply _component_design/Eyebrow.md]: updated across doc.

File: _component_design/Eyebrow.md (hunk 26 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: updated

File: _component_design/Eyebrow.md (hunk 27 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: updated.

File: _component_design/Eyebrow.md (hunk 28 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: updated across the doc.

File: _component_design/Eyebrow.md (hunk 29 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: ditto

File: _component_design/Eyebrow.md (hunk 30 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: removed.

File: _component_design/Eyebrow.md (hunk 31 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: updated across doc.

File: _component_design/Eyebrow.md (hunk 32 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: changed.

File: _component_design/Eyebrow.md (hunk 33 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: updated.

File: _component_design/Eyebrow.md (hunk 34 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: updated across doc.

File: _component_design/Eyebrow.md (hunk 35 above)
Comment (technabors) [reply _component_design/Eyebrow.md]: updated.

File: _component_design/left-nav.md (hunk 36 above)
Comment (technabors) [reply _component_design/left-nav.md]: updated

**Pull request merged**